pkg/apis/serving/v1alpha1/zz_generated.openapi.go

```diff
@@ -1,3 +1,5 @@
+// +build !ignore_autogenerated
```

Comment:
What caused this to move?

Reply:
Go coverage tests used to post some warnings messages like:
```
Starts calc.runProfiling(...)
9
2019/06/21 20:56:13 go cmdArgs=[test ./... -covermode=count -coverprofile /logs/artifacts/coverage_profile.txt]
10
2019/06/21 20:56:47 Error running ‘go test -coverprofile ’: error=‘exit status 2’; combined output=‘# github.com/knative/serving-operator/pkg/apis/serving/v1alpha1
11
pkg/apis/serving/v1alpha1/zz_generated.deepcopy.go:16: +build comment must appear before package clause and be followed by a blank line
12
pkg/apis/serving/v1alpha1/zz_generated.defaults.go:16: +build comment must appear before package clause and be followed by a blank line
13
pkg/apis/serving/v1alpha1/zz_generated.openapi.go:16: +build comment must appear before package clause and be followed by a blank line
14
? github.com/knative/serving-operator/cmd/manager [no test files]
15
? github.com/knative/serving-operator/pkg/apis [no test files]
16
’
17
2019/06/21 20:56:47 coverage profile created @ ‘/logs/artifacts/coverage_profile.txt’
```